### DISCUSSION TITLE
Qual faculdade para tecnologo  ?

### DISCUSSION THREAD

**Discussion by Lucas1906** [frontendbr/forum#809] [Discussão]
Pessoal decidi q irei fazer um curso superior...  Mesmo analisando e vendo q não é algo tao necessário para área de front end algumas coisas fizeram eu decidir em fazer... Quero algo rapido entao irei fazer um tecnólogo. Quais faculdades vcs indicariam ? andei vendo 3, FIAP, São Judas, Anhebi.  Ja q é um curso rapido to levando em consideração o nome delas no mercado e a nota no Mec. Se alguem estuda/etuou em algumas delas poderia deixa uma opinião. Obs: to quase decidindo pela FIAP pela localização e pelo nome

**Comment by ghost** [comment]
@Lucas1906 

Dos nomes citados te indicaria FIAP. Ela é uma faculdade "disruptiva" mas ao mesmo tempo é uma faculdade conservadora no que diz respeito a linguagens de programação. Você vai ver muito Java, .Net e etc porque a faculdade tem boa relação com os principais players de tecnologia (IBM, Itaú e etc).

Você vai poder ter contato com muita coisa se tiver tempo e focado nisso.

O modelo de criar uma startup no final do curso, o modelo de não ter disciplinas semestrais e sim anuais enfim, são bem diferentes da maioria das faculdades. Mas é bem bacana, é um modelo que funciona bem.

Espero que possa ter te ajudado.

**Reply by marcokoyama** [reply]
Extremamente cara!

**Comment by felipefialho** [comment]
Complementando: #39 #653 #518

**Comment by florbraz** [comment]
A sua idéia é que seja algo realmente rápido? Ou é, de fato, ganhar mais
conhecimento?

Falando da miinha experiência, eu fiz um tecnólogo (Sistemas para a
Internet) e hoje em dia penso muito que eu deveria ter cursado ciência da
computação.

Primeiro, porque, por já trabalhar na área há bastante tempo, não aprendi
muita coisa no tecnólogo - acabou sendo muito mais um diploma do que
aprendizado real. Não sei se seria assim em toda faculdade, mas dos relatos
que já ouvi, é meio que a regra - um curso mais raso, para abordar diversos
assuntos práticos, porém com pouca base teórica.

Segundo, porque, mesmo não sendo explícito no início, a base teórica te dá
insumos para ir mais facilmente para qualquer lado dentro da programação,
qualquer linguagem ou paradigma se torna muito mais palpável com uma boa
base teórica.

[]'s
Flor Braz
*http://florbraz.com <http://florbraz.com/>*

**Comment by nathpaiva** [comment]
Eu faria [Engenharia de Software](https://www.unicesumar.edu.br/ead/cursos-graduacao/engenharia-de-software/)

**Comment by MathCunhaDev** [comment]
@Lucas1906 

Já passei por Analise e desenvolvimento de Sistemas na Unip e também na Faculdade Impacta que é focada em tecnologia, na Unip foi tudo uma decepção e na Impacta foi ótimo, faculdade com estrutura e focada em conhecimento prático com apresentações e eventos, ano passado rolou o Google Dev Fest lá e foi bem legal. 

Hoje por motivos de grana e morar sozinho (não tá fácil), estou migrando pra Fatec SP em ADS também, possui uma base teórica extensa e pode ser comparado a um (quase) barachel por que são 4 anos. Conversando com um amigo que é dev no Canadá e que fez Fatec tbm, ele disse que foi muito fácil reconhecer o diploma e arranjar trabalho. 

Acho que você deve escolher não por ser rápido e etc, mas pelo que você pretende do seu futuro.

Abçs

**Comment by silvaMatheus** [comment]
Alguem mais ja estudou/estuda na impacta ? oq acham ?

**Comment by olegon** [comment]
Uma opção é a FATEC _(Faculdade de Tecnologia)_, uma faculdade pública que oferece cursos tecnólogos. Aproveita que as [incrições](https://www.vestibularfatec.com.br/home/) para o vestibular estão abertas.

**Comment by lidimayra** [comment]
Sou formada em ADS pela FATEC Jundiaí. O curso foi excelente para o meu crescimento em diversos aspectos. Desenvolvimento de competências técnicas e generalistas, aprendizagem baseada em projetos, possibilidade de atuar com pesquisa científica (Iniciação Científica e Tecnológica e Jornada de Pesquisa), diversas atividades de extensão (participei do Clube de Arduino, desenvolvimento mobile, entre outros. Há também oficina de games e Linux), participação em eventos (Semana de Tecnologia)... enfim, uma série de fatores que superaram as expectativas que eu tinha em relação à faculdade. Recomendo fortemente! :smiley:

**Comment by MathCunhaDev** [comment]
@lidimayra 

Lidiane, você sabe dizer se os cursos da Fatec são reconhecidos ou podem ser validados internacionalmente? Tenho o sonho de trabalhar no Canadá e isso seria o ideal. 

Att.

**Reply by lidimayra** [reply]
Voltando a essa issue 4 anos depois porque ela apareceu nas minhas notificações.

E hoje posso falar mais sobre isso, porque eu acabei saindo do Brasil, hoje trabalho na Alemanha... eu tive problemas com o meu diploma para conseguir o Blue Card, porque embora eu tivesse todos os pré-requisitos descritos nos processos alemães e um doc de aprovação para o Blue Card emitido pela agência de trabalho alemã, o consulado da Alemanha no Brasil disse que não emitem Blue Cards para tecnólogos, independente da faculdade. Consegui o work permit, no entanto, que é o visto para trabalhador qualificado e foi com ele que eu reloquei.

Então é aquilo lá, né, acho que muda muito de país pra país. Não sei como seria no Canadá.

A essa altura do campeonato, imagino que essa informação não acrescente muita coisa pra você, depois de tanto tempo. Mas achei que fazia sentido relatar aqui caso alguém esteja na mesma situação.

**Comment by kelciocajueiro** [comment]
@MatCunha você poderá fazer a equivalência do seu diploma na WES para ele ser válido para imigração canadense. Se seu curso for de 2 anos e meio provavelmente ele será enquadrado como "two-year post-secondary credential".

**Comment by lidimayra** [comment]
Oi @MatCunha!!
Os cursos da FATEC são de ensino superior, tem duração de 6 semestres e são reconhecidos pelo MEC. Desta forma, para revalidar o diploma no exterior é necessário verificar quais são as exigências específicas do país que você deseja.

Aliás, tenho amigos que fizeram graduação sanduíche, na qual cursaram parte do curso na FATEC e parte do curso em universidades estrangeiras (como dá pra ver [aqui](http://buscatextual.cnpq.br/buscatextual/visualizacv.do?id=K4646737T2)  e [aqui](http://buscatextual.cnpq.br/buscatextual/visualizacv.do?id=K4959206U5)).

**Comment by felpsg** [comment]
Olá @lidimayra tudo bem?

Poderia tirar algumas dúvidas com você a respeito do seu work na Alemanha?
Se não for incomodo, claro!

Abs.

**Reply by lidimayra** [reply]
Oi @felpsg , claro, no que eu puder ajudar, fico à disposição!!
Pode compartilhar as dúvidas aqui mesmo, ou se preferir pode me enviar um email também: lidi.mayra@gmail.com

**Comment by FelipeBaamonde** [comment]
passei na fatec e na sptech(privada), na sptech sao 2 anos e tem um estagio garantido ja no 2 semestre, alem do 1 semestre ser gratuito e começar a pagar a mensalidade a partir do segundo com a grana do estagio.
a fatec é 3 anos, porem é publica.
OBS: estou matriculado na UNIVESP uma faculdade ead publica, entao n poderei fazer a fatec ao msm tempo q ela, já com a sptech eu consigo.
